directives/_static/css/aplus_tabs.css

```diff
@@ -0,0 +1,71 @@
+@import url("default.css");
```

Comment:
I don't think this file should import the CSS styles of the whole aplus theme. Isn't the purpose of this aplus_tabs.css file to only contain the rules for the content tabs and these styles are only included in the A+ chapter if tabs are used there?

Reply:
Yes, I think I was doing some testing and I forgot to remove that import